configure.ac

```diff
@@ -202,13 +202,21 @@ AC_SUBST(TEST_FORTRAN_NETCDF) TEST_FORTRAN_NETCDF="yes"
 
 AC_ARG_ENABLE([netcdf],
               [AS_HELP_STRING([--enable-netcdf],
-                              [Build HDF4 versions of NetCDF APIs (version 2.3.2) [default=yes]])],,
+                              [HDF4 builds replacement netCDF 2.3.2 API calls that are exported (e.g., ncopen()).
+                               Enabling this option uses the original netCDF names when building these functions.
+                               Disable prefixes them with 'sd_' to avoid name clashes.
+                               This does NOT affect building the HDF4-built netCDF tools ncdump and ncgen,
+                               nor does it disable any HDF4 functionality - this option only affects
+                               API call names.
+                               Disabling this option disables building the netCDF Fortran wrappers (they
+                               will not be built with either the original or sd_ prefixed names).
+                               [default=yes (HDF4-built netCDF API calls will be exported undecorated)]])],,
```

Comment:
This is such a confusing option. After 4.2.16 releases, I'll do some internal renaming to make the purpose of this option clearer to maintainers.

Reply:
It's not really "enabling" functionality, it's just controlling name-mangling, so "enable netCDF" is a terrible name.